.pre-commit-config.yaml

```diff
@@ -26,7 +26,8 @@ repos:
       # args: [--show-error-codes]
       additional_dependencies: [
         "boto3-stubs[s3,logs]",
-        pandas-stubs,
+        pandas==1.5.*,
+        pandas-stubs==1.5.*,
```

Comment:
nit: fairly unlikely that pandas will release another 1.5.x at this point, but I think this should actually be like so:

```suggestion
        pandas==1.5.3,
        pandas-stubs==1.5.3.*,
```

because we are definitely using pandas 1.5.3 as per our frozen requirements

Reply:
And then we need to also pin that stub version in requirements-dev.in

Reply:
Thought you'd push back on adding the `pandas` as a dependency on the pre commit hook.

Reply:
No, I think it's probably going to end up being the right thing to do - I suspect we probably just need to have it install all of our dependencies...

Reply:
Well.. If you couldn't solve it in 5 mins. It's impossible to do. 😜

Reply:
😭